### ISSUE TITLE
Neighbor attributes are private

### ISSUE THREAD

**Issue by advancedlogic** [ynqa/wego#47]
Could you make Neighbor word and similarity accessible?

**Comment by ynqa** [comment]
@advancedlogic Excuse me, what do you mean?

I've already introduced similarity search mechanisms:
- https://github.com/ynqa/wego/blob/master/search/README.md

**Comment by advancedlogic** [comment]
If I use your Searcher programmatically, once I have a list of neighbors I do not have access to the attributes word and similarity because they are private inside the struct (lower case) and I do not have any get/set accessors to get their value

**Comment by ynqa** [comment]
Right.. current implementations are a little terrible to utilize any other cases. Anyway, it should remove `io.Reader` from `Searcher` structure. And also please tell me your use cases, and which I utilize for other improvements.

**Comment by ynqa** [comment]
@advancedlogic Right.. current implementations are a little terrible to utilize except for internal usages. Anyway, it should remove io.Reader from Searcher structure. And also please tell me your use cases, and which I will utilize for improvements.
